packages/fx-core/src/common/local/localCertificateManager.ts

```diff
@@ -90,22 +90,25 @@ export class LocalCertificateManager {
         }
       }
 
+      localCert.found = !!certThumbprint;
       if (!certThumbprint) {
         if (failIfNotFound) {
-          localCert.found = false;
           return localCert;
         }
         // generate cert and key
         certThumbprint = await this.generateCertificate(certFilePath, keyFilePath);
       }
 
       if (needTrust) {
-        if (certThumbprint && (await this.verifyCertificateInStore(certThumbprint))) {
+        if (await this.verifyCertificateInStore(certThumbprint)) {
           // already trusted
           localCert.isTrusted = true;
           localCert.alreadyTrusted = true;
         } else {
           localCert.alreadyTrusted = false;
+          if (failIfNotFound) {
```

Comment:
It seems the param name does not match the case, how about:
- renaming to `readonly` (indicate the setup is a read-only operation, no generate and no trust)?
- or, adding another optional param `failIfNotVerified`?

Reply:
renamed by: checkOnly